HaskellDates.hsproj/HaskellDates.hs

```diff
@@ -0,0 +1,62 @@
+import Data.Time
+import Data.Time.Calendar.OrdinalDate
+import Control.Applicative
+
+data Event = Event {
+  date :: Day,
+  daysOfWeek :: [Int],
+  week :: Int
+}
+
+-- WeekDay for debugging purposes
+newtype WeekDay = WeekDay Int
```

Comment:
It may be worth making this into a sum type, like `data WeekDay = Sunday | Monday | Tuesday | ...`. You can derive `Ord` and `Show` instances that should remove most of the tedium of `Int` conversion, and could provide better semantics and type safety.